centos7.spec

```diff
@@ -0,0 +1,37 @@
+%global pgmajorversion 12
+%global pginstdir /usr/pgsql-%{pgmajorversion}
+
+Summary:        Postgres AGE
+Name:           postgres-%{pgmajorversion}-age
+Version:        1
+Release:        %{dist}
+License:        Apache
+Group:          Databases
+Url:            https://github.com/apache/age.git
+Source0:        %{name}-%{version}.tar.gz
+BuildRoot:      %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+
+%description
+This package provides extensions for Apache Age
```

Comment:
Add a bit details in description like bellow:
Apache AGE is an extension for PostgreSQL that enables users to leverage a graph database on top of the existing relational databases.